storage_service/locations/api/resources.py

```diff
@@ -272,6 +276,26 @@ def get_objects(self, space, path):
         message = _('This method should be accessed via a versioned subclass')
         raise NotImplementedError(message)
 
+    def default(self, request, **kwargs):
```

Comment:
This is the view for processing a request to `/api/v2/location/default/$purpose/`. It attempts to find the default location for that purpose and redirect the client to it, e.g. `/api/v2/location/<uuid>/`.